astrokat/test/testutils.py

```diff
@@ -0,0 +1,88 @@
+import logging
+import os
+from six.moves import StringIO
+
+from astrokat import observe_main, simulate, utility
+
+
+__all__ = ['yaml_path', 'LoggedTelescope']
+
+PROPOSAL_ID = "CAM_AstroKAT_UnitTest"
+OBSERVER = "KAT Tester"
+
+
+def yaml_path(file_path):
+    """Convenient method for finding the yaml's file absolute path.
+
+    Args:
+        file_path (str): YAML file path
+
+    Returns:
+        str: YAML file absolute path
+    """
+    tests_path = os.path.abspath(os.path.dirname(__file__))
+    yaml_file = os.path.abspath(os.path.join(tests_path, file_path))
+    assert os.path.isfile(yaml_file)
+    return yaml_file
+
+
+def extract_start_time(yaml_file):
+    """ Convenient method to extract start_time from yaml
+
+    :param yaml_file: full path file name to yaml file
+    :return: start_time if it exists in yaml file
+    """
+    yaml = utility.read_yaml(yaml_file)
+    if yaml and yaml.get('durations') and yaml.get('durations').get('start_time'):
+        return yaml['durations']['start_time']
+
+
+def execute_observe_main(file_name):
+    """ Convenient method to run observer_main with correct parameters
+
+    :param file_name: relative path to yaml file
+    """
+    yaml_file = yaml_path(file_name)
+    start_time = extract_start_time(yaml_file)
+
+    params = [
+        '--yaml', yaml_file,
+        '--observer', OBSERVER,
+        '--proposal-id', PROPOSAL_ID,
+        '--dry-run',
+        '--sb-id-code', os.getenv('SB_ID_CODE'),
```

Comment:
As noted above, doesn't make sense to pass `None` as a command line arg, if environment variable not defined.